sdk/tables/azure-data-tables/inc/azure/data/tables/table_audience.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#pragma once
+

```

Comment:
For `std::string` and `std::move()`
```suggestion

#include <string>
#include <utility>

```